src/api/python/cvc5.pxi

```diff
@@ -1698,6 +1712,49 @@ cdef class Term:
         term.cterm = get2(t)
         return (get0(t), get1(t), term)
 
+    def isSetValue(self):
+        return self.cterm.isSetValue()
+
+    def getSetValue(self):
+        elems = []
+        for e in self.cterm.getSetValue():
+            term = Term(self.solver)
+            term.cterm = e
+            elems.append(term)
+        return set(elems)
+
+    def isSequenceValue(self):
+        return self.cterm.isSequenceValue()
+
+    def getSequenceValue(self):
+        elems = []
+        for e in self.cterm.getSequenceValue():
+            term = Term(self.solver)
+            term.cterm = e
+            elems.append(term)
```

Comment:
I've seen this pattern over and over again. Can we add a constructor to term to allow for `elems.append(Term(self.solver, e))`?
And then have `return [Term(self.solver, e) for e in self.cterm.getSequenceValue()]`?